### PR TITLE
Change/copyedit to "Context Lines” wording in UI

### PR DESCRIPTION
**Pull request by hughlilly**
This PR suggests some changes to the wording in the UI of the “Context lines” feature in Search editors. Though the underlying variables are named in line with the idea of the displayed lines providing context to the user, I think the wording I’ve chosen here (“surrounding lines”) is more natural and makes more sense in terms of UI writing.

### CODE REVIEW

File: src/vs/workbench/contrib/search/browser/searchWidget.ts [hunk 1]
```diff
@@ -364,7 +364,7 @@ export class SearchWidget extends Widget {
 
 		this.showContextToggle = new Toggle({
 			isChecked: false,
-			title: appendKeyBindingLabel(nls.localize('showContext', "Toggle Context Lines"), this.keybindingService.lookupKeybinding(ToggleSearchEditorContextLinesCommandId), this.keybindingService),
+			title: appendKeyBindingLabel(nls.localize('showContext', "Show/hide surrounding lines"), this.keybindingService.lookupKeybinding(ToggleSearchEditorContextLinesCommandId), this.keybindingService),
```
Comment (andreamah) [inline src/vs/workbench/contrib/search/browser/searchWidget.ts:367]: Please capitalize instances of `Show/hide surrounding lines` (`Show/Hide Surrounding Lines`)

**Review by andreamah — CHANGES_REQUESTED** [review]
Apologies for getting to this so late! I agree with the change to "surrounding lines". Also, would you be able to create an associated issue with this? Thanks!

**Comment by roblourens** [comment]
I don't have a strong opinion one way or the other, but what's wrong with "context lines"? I think this is popular terminology, for example it's used in grep.

**Comment by andreamah** [comment]
> I don't have a strong opinion one way or the other, but what's wrong with "context lines"? I think this is popular terminology, for example it's used in grep.

Do we document anywhere what 'context lines' are? I hadn't run into the term before and I was a bit confused by it

**Comment by roblourens** [comment]
Seems that it's mentioned here: https://code.visualstudio.com/docs/editor/codebasics#_search-editor

But it's up to you, whatever you think is better wording

**Comment by andreamah** [comment]
Looking into this a little bit more, it seems that context lines is actually used in a few places already. I was originally afraid that using the term 'context lines' was just sort of exposing a term that we use in implementation. Closing for now.

**Pull request closed without merging**